Packs/Silverfort/Integrations/Silverfort/Silverfort.py

```diff
@@ -196,13 +208,18 @@ def main():
         PARSE AND VALIDATE INTEGRATION PARAMS
```

Comment:
```suggestion
def main():  # pragma: no cover
    """
        PARSE AND VALIDATE INTEGRATION PARAMS
```

Reply:
Looks like the `main` function being uncovered drops [the coverage](https://output.circle-artifacts.com/output/job/2e0b4910-197b-415a-a54b-001c7aec054b/artifacts/0/artifacts/coverage_report/html/d_16421cef7a32524b_Silverfort_py.html) significantly.

@liavSilverfort - Please make the change above to ignore the `main` function from being calculated in coverage. That should take care of the UTs failing.